package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kisskissbankbank/kitten",
-  "version": "8.3.1",
+  "version": "8.4.1",
```

On est plutôt sur la 8.4.0 il me semble

```suggestion
  "version": "8.4.0",
```